### PR TITLE
Periodic Customer Spawning

### PR DESCRIPTION
**Pull request by rjcayaco**

### CODE REVIEW

**Review by teanbes — CHANGES_REQUESTED** [review]
which PR should we merge?

**Review by teanbes — CHANGES_REQUESTED** [review]
which PR should we merge?

**Review by jeffrey-tong — APPROVED** [review]

File: Assets/Development/Scripts/AI/C_Manager.cs [hunk 1]
```diff
@@ -0,0 +1,161 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class C_Manager : MonoBehaviour
+{
+    public Transform Counter;
```
Comment (jeffrey-tong) [inline Assets/Development/Scripts/AI/C_Manager.cs:8]: Please change these to [SerializeField] private if possible

**Review by teanbes — APPROVED** [review]

**Review by jeffrey-tong — COMMENTED** [review]
A lot of your variable declarations are public, please change those to [SerializeField] private where possible, and try to make more of the private if u can. If you cant get those changes done by tuesday that's ok.

**Pull request merged**